Sources/BrowserServicesKit/PrivacyConfig/PrivacyConfigurationManager.swift

```diff
@@ -27,9 +27,13 @@ public protocol EmbeddedDataProvider {
 }
 
 public protocol PrivacyConfigurationManaging {
-    
+    var embeddedConfigData: PrivacyConfigurationManager.ConfigurationData { get }
+    var fetchedConfigData: PrivacyConfigurationManager.ConfigurationData? { get }
```

Comment:
I wonder if we can get rid of that? Based on the fact that reload(nil) should fetch embedded, and reload(etag, data) would use provided one, maybe we can simply tweak tests so these become private?